src/registrar/forms/application_wizard.py

```diff
@@ -153,7 +153,8 @@ def from_database(cls, obj: DomainApplication, join: str, on_fetch: Callable):
 
 class OrganizationTypeForm(RegistrarForm):
     organization_type = forms.ChoiceField(
-        choices=DomainApplication.OrganizationChoices.choices,
+        # use the long names in the application form
+        choices=DomainApplication.OrganizationChoicesVerbose.choices,
```

Comment:
(Q) do we still need this after the change? Its looks like we probably do but just to make sure its not a leftover

Reply:
yep